tests/reboot/reboot_test.go

```diff
@@ -37,8 +37,8 @@ var _ = Describe("{RebootOneNode}", func() {
 		}
 
 		Step("get nodes for all apps in test and reboot their nodes", func() {
+			var nodesToReboot []node.Node
 			for _, ctx := range contexts {
-				var nodesToReboot []node.Node
 
 				Step(fmt.Sprintf("get nodes for %s app to reboot, where volumes are attached", ctx.App.Key), func() {
 					volumes, err := Inst().S.GetVolumes(ctx)
```

Comment:
as mentioned before, as we're stopping  a single node at a time, this logic can go away, and replace for `node.GetWorkerNodes()`

Reply:
incorporated review comment